src/refiners/fluxion/utils.py

```diff
@@ -34,6 +34,31 @@ def interpolate(x: Tensor, factor: float | torch.Size, mode: str = "nearest") ->
     )  # type: ignore
 
 
+# Adapted from https://github.com/pytorch/vision/blob/main/torchvision/transforms/_functional_tensor.py
+def normalize(tensor: Tensor, mean: list[float], std: list[float], inplace: bool = False) -> Tensor:
+    assert tensor.is_floating_point()
+    assert tensor.ndim >= 3
+
+    if not inplace:
+        tensor = tensor.clone()
+
+    dtype = tensor.dtype
+
+    mean_tensor = as_tensor(mean, dtype=tensor.dtype, device=tensor.device)
```

Comment:
why using the as_tensor and not just the standard way to create a tensor? (data is a list of float here)